frontend/app/wiring/AppComponents.scala

```diff
@@ -0,0 +1,96 @@
+package wiring
+
+import actions.TouchpointOAuthActions
+import play.api.routing.Router
+import play.api.libs.ws.ahc.AhcWSComponents
+import play.api.mvc.EssentialFilter
+import play.api.BuiltInComponentsFromContext
+import com.softwaremill.macwire.wire
```

Comment:
I didn't use wire in subs frontend (or @pvighi  in members data api) as I'm not sure it buys us much for an extra dependency.  I'm guessing intellij would find and check everything still as well as in SBT?